functests/ocs/data_validation_test.go

```diff
@@ -1,30 +1,27 @@
-package functests_test
+package ocs_test
 
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	tests "github.com/openshift/ocs-operator/functests"
-	deploymanager "github.com/openshift/ocs-operator/pkg/deploy-manager"
+
 	k8sbatchv1 "k8s.io/api/batch/v1"
 	k8sv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
 )
 
 var _ = Describe("job creation", DataValidationTest)
 
 func DataValidationTest() {
-	var k8sClient *kubernetes.Clientset
-	var deployManager *deploymanager.DeployManager
-
-	BeforeEach(func() {
-		RegisterFailHandler(Fail)
-		deployManager, err := deploymanager.NewDeployManager()
-		Expect(err).To(BeNil())
+	dm := tests.DeployManager
```

Comment:
If we are using a global DeployManager what is the need to assign it to a different local variable?
We should be able to use the global var directly right? Since we don't even mutate it.